ios/RCTWebRTC/RTCVideoViewManager.m

```diff
@@ -87,8 +87,6 @@ @implementation RTCVideoView {
  * Tells this view that its window object changed.
  */
 - (void)didMoveToWindow {
-  [super didMoveToWindow];
```

Comment:
asking as I'm not an iOS expert yet: why is there no need to call super exactly ?

Reply:
is there some magic there that would do it automatically ? or is the `UIView` just an interface/empty class ?

Reply:
According to the documentation there is no need to call it.